Generic sql bug mssql driver (#23797)
demisto/content: Packs/GenericSQL/ReleaseNotes/1_0_13.md

#### Integrations
##### Generic SQL

> - Fixed an issue with 'Microsoft SQL Server - MS ODBC Driver' had connection error.

Reviewer: ```suggestion
- Fixed an issue where the integration failed to connect to Microsoft SQL Server.
```

> - Fixed an issue with 'Microsoft SQL Server  - MS ODBC Driver' had connection error.

Reviewer 2: ```suggestion
- Fixed an issue with 'Microsoft SQL Server - MS ODBC Driver' had connection error.
```
